check-services.php

```diff
@@ -61,10 +66,68 @@
     }
 }
 
-$sql = 'SELECT * FROM `devices` AS D, `services` AS S WHERE S.device_id = D.device_id ' . $where . ' ORDER by D.device_id DESC';
+if (!isset($query)) {
+    $query = "SELECT * FROM `devices` WHERE `disabled` = 0 $where ORDER BY `device_id` ASC";
+}
+
+foreach (dbFetch($query) as $device) {
```

Comment:
I added device loop because of the RunRules function below, so it would execute the function after polling all services from one device before continuing on to the next device and it's services.

Reply:
Reverted back to one loop.